.github/workflows/mirror-changelog.yml

```diff
@@ -0,0 +1,134 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Mirror Toolbox Changelog
+
+on:
+  pull_request_target:
+    types: [opened, edited]
+
+jobs:
+  add-release-notes:
+    if: github.actor == 'renovate[bot]' && startsWith(github.head_ref, 'renovate/googleapis-genai-toolbox')
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - name: Add Toolbox Release Notes to PR Body
+        uses: actions/github-script@v6
+        env:
+          REQUIRED_KEYWORDS: 'mssql'
```

Comment:
Could we support multiple keywords? Future update: I think this will cover this but we may also want to support sql-server